build/lender/Lender.sol

```diff
@@ -389,16 +392,30 @@ contract Lender {
   function withdraw(address e) external authorized(admin) returns (bool) {
     // Get the token to be withdrawn
     IErc20 token = IErc20(e);
-    
-    // Transfer the accumulated fees to the admin
-    Safe.transfer(token, admin, fees[e]);
+
+    // Get the balance to be transferred
+    uint256 balance = fees[e];
 
     // Reset accumulated fees of the token to 0
     fees[e] = 0;
+    
+    // Transfer the accumulated fees to the admin
+    Safe.transfer(token, admin, balance);
 
     return true;
   }
 
+  /// @notice This method returns the fee based on the amount passed to it. If
+  /// feenominator is 0, then there is no fee.
+  /// @param a Amount of underlying tokens to calculate the fee for
+  /// @return uint256 The total for for the given amount
+  function calculateFee(uint256 a) internal view returns (uint256) {
+    if (feenominator == 0) {
+      return 0;
+    } 
+    return a / feenominator;
```

Comment:
solidity does have a ternary, use it

`return feenominator > 0 ? a / feenominator : 0`

also can this internal function be `pure` vs `view` ? do it if so.